python_scripts/read_binary_output.py

```diff
@@ -324,6 +324,10 @@ def _read_binary_block_v8(bfile):
     """Read one output block from SMASH binary file."""
     return _read_binary_block_v7(bfile) # Nothing has changed for non-extended
 
+def _read_binary_block_v9(bfile):
+    """Read one output block from SMASH binary file."""
+    return _read_binary_block_v7(bfile) # Nothing has changed for non-extended
```

Comment:
This should be v9

Reply:
I think this is correct. The `_read_binary_block_v9` function should do the same as the function directly above and call the `_read_binary_block_v7` function. The latter is used for the binary reader in the case of a non extended output format, which hasn't changed since output version 7.
For the new version 9 extended format, there is a new function added.

Reply:
True.